settings.gradle

```diff
@@ -1,6 +1,7 @@
 rootProject.name = 'parseq'
 
 def modules = [ /* the name of the modules to use */
+    'parseq', // core part code, including test utility
```

Comment:
Does this still include test utility? The comment seems indicating that.

Reply:
True, this is no longer accurate. I'll fix this